docs/content-template-for-developers.md

```diff
@@ -0,0 +1,84 @@
+# Overview 
```

Comment:
Add the header part as:

---
title:  "Content Template for Developers "
source_url: 'https://github.com/SenseNet/sensenet/blob/master/docs/content-template-for-developers.md'
category: Developers
version: v6.0
tags: [content, content template]
---